test/user.test.js

```diff
@@ -1977,6 +1977,43 @@ describe('User', function() {
         ], done);
       });
 
+      it('keeps sessions AS IS if non-email property is changed using updateAll', function(done) {
+        var userPartial;
+        async.series([
+          function createPartialUser(next) {
+            User.create(
+                   { email: 'partial@example.com', password: 'pass1', age: 25 },
```

Comment:
Nitpick: use two spaces for indentation. I am surprised `eslint` is not complaining.